bun-tests/fake-snippets-api/routes/snippets/images.test.ts

```diff
@@ -0,0 +1,39 @@
+import { expect, test } from "bun:test"
+import { getTestServer } from "bun-tests/fake-snippets-api/fixtures/get-test-server"
+
+test("get schematic svg of a snippet", async () => {
+  const { axios, seed } = await getTestServer()
+
+  const addedSnippet = seed.snippets[0]
+
+  const snippetName = addedSnippet.name.replace(
+    addedSnippet.owner_name + "/",
+    "",
+  )
+
+  const response = await axios.get(
+    `/api/snippets/images/${addedSnippet.owner_name}/${snippetName}/schematic.svg`,
```

Comment:
When this route is used it shouldnt return json, just the svg content. That way it can be opened in a browser